school/templates/index.html

```diff
@@ -0,0 +1,42 @@
+{% extends "school_base.html" %}
```

Comment:
This is a simple UI for now - that just has a table with all data on one side and on the other side a form to create a new instance of the class. Will clean up once we get the models cleared up